CHANGELOG.md

```diff
@@ -38,7 +38,7 @@
     * Avoid re-running all codeaction requests at low priority (PR: [#74083](https://github.com/dotnet/roslyn/pull/74083))
     * Reduce time spent in ConflictResolver.Session.GetNodesOrTokensToCheckForConflicts (PR: [#74101](https://github.com/dotnet/roslyn/pull/74101))
     * Avoid allocations in AbstractSyntaxIndex<>.GetIndexAsync( PR: [#74075](https://github.com/dotnet/roslyn/pull/74075))
-* Bump xamltools to 17.12.35112.24 (PR: [#7309](https://github.com/dotnet/vscode-csharp/pull/7334))
+* Bump xamltools to 17.12.35117.32 (PR: [#7362](https://github.com/dotnet/vscode-csharp/pull/7362))
```

Comment:
Instead of modifying this, can add an entry to the latest section with these changes?  2.39 is already snapped.

Reply:
Sorry! didn't see that it was in the 2.39 section.